### ISSUE TITLE
Integrate Netlify CMS to Site (mf.com)

### ISSUE THREAD

**Issue by budparr** [theNewDynamic/mfocus-tracking#3]
This is a  <del>Jekyll</del> **Hugo** site so we <del>can't</del> **can** use our pre-built module. 

- [x] Set up Netlify CMS fields
- [ ] Set up Auth
- [x] Test

Need to know:

- [ ] Do we use Editorial Workflow? (previews)

I believe this site is not on Netlify, so we'll need auth information, so this issue is dependent upon #2 being completed.

Site: https://github.com/missionfocus/www.missionfocus.com

**Comment by regisphilibert** [comment]
> This is a Jekyll site so we can't use our pre-built module.

We could if we host the CMS on a different domain. We'd build a Hugo site just to host the CMS but could use our module for ease of configuration.

It's to investigate of course. But we could at least easily maintain the yaml file this way...

**Comment by andreweick** [comment]
I think this should be a lower priority (netlify CMS) until we figure out if (we) want to stay on Jekyll or migrate to Hugo.  I need to get with the boss to see if she wants to re-work the site right now.

It is on netlify.

**Comment by andreweick** [comment]
(enabled git-gateway)

**Comment by andreweick** [comment]
When you(s-all) have time, on the road map is this site migration to Hugo.  We could delay the CMS integration until after the Hugo migration if that works better

**Comment by regisphilibert** [comment]
> When you(s-all) have time, on the road map is this site migration to Hugo.

You mean eick.com?

**Comment by andreweick** [comment]
No, This issue is for https://missionfocus.com

**Comment by regisphilibert** [comment]
@andreweick you can take a look at this PR: https://github.com/missionfocus/www.missionfocus.com/pull/3 and its deploy preview: https://deploy-preview-3--www-missionfocus-com.netlify.app/cms/

I suppose you have to apply the same Netlify Auth settings you did for eick.com in order to be able to log in to the `/cms` page. Any edits you do will be applied to the `add-netlifycms` branch and trigger a new deploy on the URL shared above.

If you're happy with it, we'll set this to apply to `master`, and merge. Thanks.

To try it on your local, pull the branch, hit `npm install` and then `npm run dev:cms` this will serve Hugo and the CMS at `http://localhost:1313/cms/`

**Comment by budparr** [comment]
@andreweick Is this ready to go?

**Comment by andreweick** [comment]
@budparr yes, / close it -- I'll get auth working separately
